### DISCUSSION TITLE
Uploading to SSOP20(mini clone) with uno clone instead of FTDI

### DISCUSSION THREAD

**Discussion by Gestreleth** [dbuezas/lgt8fx#149] [Q&A]
I was dumb enough not to order an FTDI so I found out that apparently an UNO can be used instead. Apparently you can do so by connecting the UNO's reset pin to ground, and connecting the UNO's ground, 5v, TX and RX pins accordingly to the destination board. This doesn't work with the green mini board I'm trying to upload to, only resulting in this:

```
Arduino: 1.8.12 (Windows 10), Board: "LGT8F328, Default (64), Internal, 16 MHz, 328P-SSOP20 (e.g. green pseudo pro mini)"

Sketch uses 1100 bytes (3%) of program storage space. Maximum is 29696 bytes.
Global variables use 9 bytes of dynamic memory.
An error occurred while uploading the sketch
avrdude: stk500_getsync() attempt 1 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 2 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 3 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 4 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 5 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 6 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 7 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 8 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 9 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 10 of 10: not in sync: resp=0x00
Could not find boards.txt in C:\Program Files (x86)\Arduino\hardware\lgt8fx-master\docs. Is it pre-1.5?
Could not find boards.txt in C:\Program Files (x86)\Arduino\hardware\lgt8fx-master\docs. Is it pre-1.5?

This report would have more information with
"Show verbose output during compilation"
option enabled in File -> Preferences.
```

In addition, the TX and RX lights on the UNO flash at around 1 second intervals, as does the blue LED on the mini clone. Is it possible to use an Uno board to upload to the green pin or should I just order an FTDI and shelf the mini clone for now?

**Comment by youxiaojie** [comment]
the uno' serial port are taken by usb. when comminicate with pc.  and what is the baud of usb ttl(32u2 as converter)? and howto distinguish which bootloader the lgt328d on serial port or atmaga328pu on uno because they are connected to.the same txrx

**Comment by dbuezas** [comment]
On top of what you are doing, try reseting the mini board when the upload begins. And make sure the uno has an empty sketch that doesn't use serial.
The only other option without an ftdi is to use the uno as ISP.

**Comment by dbuezas** [comment]
Did it work? I'll move this to a discussion

**Reply by jokokiswan** [reply]
i tried lgfp328 promini version, using usb to tll (just like bare esp12E) to upload firmware, resetting on upload, it did not work? any solution?

promini--------usb2ttl (prolific usb)
rx--------------tx
tx--------------rx
gnd-----------gnd
vcc------------5v

it flashing light on both usb2ttl rx tx pin led after blue light on romini flash.

IDE -> arduino IDE

**Reply by jokokiswan** [reply]
i'll try using larduino isp, thank you.
https://ett.co.th/prodAVR/NANO-F328-C/LGT8F328P.pdf
